src/common/StatsCard/StatsCard.js

```diff
@@ -30,11 +33,16 @@ const StatsCard = ({ children, className = '', onClick = () => {} }) => {
   )
 }
 
-StatsCard.Header = ({ children = null, icon = '', iconClass = '', title = '' }) => {
+StatsCard.Header = ({ children = null, icon = '', iconClass = '', title = '', tip = '' }) => {
+  const titleClass = classNames('stats-card__title', icon && 'stats-card__title--with-icon')
+
   return (
     <div className="stats-card__row">
-      {icon && <i className={iconClass}>{icon}</i>}
-      {title && <h5 className="stats-card__title">{title}</h5>}
+      <div className={titleClass}>
+        {icon && <i className={iconClass}>{icon}</i>}
+        {title && <span>{title}</span>}
+        {tip && <Tip className="project-data-card__header-tip" text={tip} />}
```

Comment:
StatsCard is a common component. You can't use specific class `project-data-card...`